### DISCUSSION TITLE
config file example for Raspberry Pi + Raspberry Cam

### DISCUSSION THREAD

**Discussion by sroller** [Motion-Project/motionplus#106] [Q&A] [answered]
I recently updated my Pi with the Debian-Bookworm distro. My Motion installation stopped working. Even with the installation guide I couldn't solve the problem.
Now I switched to Motion+ hoping the more modern approach will make things easier.
Would it be too much to ask for somebody to post an - almost - working config file for the Pi+Picam combination? I would think this is a very common hardware for this project and it would make it easier to get quick success.

Any takers?

**Comment by axeljerabek** [comment]
Hi Steffen, it is probably the complications with libcamera. I have a solution where I split up the tasks. You might have a look here:
https://www.wetransco.de/debian-bullseye-libcamera-and-motion-on-a-raspberry-pi-with-the-new-camera-module-v3-a-great-workaround/

**Reply by sroller** [reply]
thank you. That sounds like a workable workaround :-). I'll look into it. I'd love to find a real solution which works "natively" with Motion+.

**Comment by Mr-Dave** [comment, marked as answer]
In a camera.conf file of Motionplus, just specify [libcam_device camera0](https://motion-project.github.io/motionplus_config.html#libcam_device).  All libcamera parameters can be specified using the [libcam_params](https://motion-project.github.io/motionplus_config.html#libcam_params).  (No specific params are required.)

**Reply by sroller** [reply]
I understand that I need now two config files for motion+ instead of one for motion?

motionplus.conf
```
log_level 8
target_dir /home/motion/output
camera /home/motion/camera.conf
```

camera.conf
```
libcam_device camera0
width 1024
height 768
text_left PI ZERO
device_name zero0
devid_id 101
movie_filename %Y%m%d-%H%M%S
```

Assuming `libcamera-hello` produces a picture then `motionplus` should work as well, right?
